pymc/tests/test_distributions_moments.py

```diff
@@ -208,6 +209,21 @@ def test_beta_moment(alpha, beta, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "n, alpha, beta, size, expected",
+    [
+        (10, 1, 1, None, 5),
+        (10, 1, 1, 5, np.full(5, 5)),
+        (10, 1, np.arange(1, 6), None, np.floor(10 / np.arange(2, 7))),
+        (10, 1, np.arange(1, 6), (2, 5), np.full((2, 5), np.floor(10 / np.arange(2, 7)))),
```

Comment:
Why is the floor needed in the expected results?

Reply:
I mentioned this in the first comment of the pull request - the get_moment method returns an integer for some reason, while the actual mean should be a float (according to the scipy.stats.betabinom). I got the impression from other pymc distributions (binomial for example) that get_moment should return an integer anyway.. let me know if this was incorrect!

Reply:
I see. I forgot there binomial part of the beta-binomial xD. Then we should add a floor or round (whichever makes more sense) as an explicit operation in the `get_moment` as well

Reply:
Noted!